src/proposals/20250529/SparkEthereum_20250529.sol

```diff
@@ -0,0 +1,239 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { AllocatorBuffer } from 'dss-allocator/src/AllocatorBuffer.sol';
+import { AllocatorVault }  from 'dss-allocator/src/AllocatorVault.sol';
+
+import { IERC20 }   from 'forge-std/interfaces/IERC20.sol';
+import { IERC4626 } from 'forge-std/interfaces/IERC4626.sol';
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+import { Optimism } from 'spark-address-registry/Optimism.sol';
+import { Unichain } from 'spark-address-registry/Unichain.sol';
+
+import { MainnetController }               from "spark-alm-controller/src/MainnetController.sol";
+import { RateLimitHelpers, RateLimitData } from "spark-alm-controller/src/RateLimitHelpers.sol";
+
+import { ISparkLendFreezerMom } from 'sparklend-freezer/interfaces/ISparkLendFreezerMom.sol';
+
+import { SparkPayloadEthereum, IEngine, EngineFlags } from "../../SparkPayloadEthereum.sol";
+
+import { IMetaMorpho, MarketParams } from 'metamorpho/interfaces/IMetaMorpho.sol';
+
+import { CCTPForwarder } from 'xchain-helpers/forwarders/CCTPForwarder.sol';
+
+interface IOptimismTokenBridge {
+    function bridgeERC20To(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes calldata _extraData
+    ) external;
+}
+
+/**
+ * @title  May 29, 2025 Spark Ethereum Proposal
+ * @notice Spark Liquidity Layer:
+ *         - Onboard Optimism, Unichain
+ *         - Mint 100m USDS into Optimism and Unichain
+ *         - Mint 100m sUSDS into Optimism and Unichain
+ *         - Increase USDe Mint and Staking Rate Limits
```

Comment:
"Swap" instead of "Staking"?

Reply:
fixed

Reply:
@supercontracts 
I mixed up two similar items. This one was correct. "Staking" should be here as it was initially.
Sorry for the confusion

Reply:
reverted back no problem